src/pages/Landing.tsx

```diff
@@ -34,7 +36,10 @@ const Tab1: React.FC = () => {
                         <IonTitle size="large">Landing</IonTitle>
                     </IonToolbar>
                 </IonHeader>
-                <ExploreContainer name="Landing page, Ratik says hello!" />
+                <div className="weTried">Video Placeholder</div>
```

Comment:
@MrClean-1 @RatikKapoor Should this div be removed and replaced with a non html element?

Reply:
Yeah definitely and let's not do classes called `weTried` that's not very maintainable

Reply:
@Special-Sharpie @Dennull

Reply:
yeah prolly should be replaced with an Ion element for typing reasons
Take a look at https://ionicframework.com/docs/api/item-divider and https://ionicframework.com/docs/api/content